upgrade-2.x.sh

```diff
@@ -467,6 +467,20 @@ else
     log "No resin-device-progress fix is required..."
 fi
 
+if version_gt "$VERSION_ID" "2.0.8" &&
+    version_gt "2.7.0" "$VERSION_ID"; then
+        log "Fixing supervisor updater..."
+        if curl --fail --silent -o "/tmp/update-resin-supervisor" https://raw.githubusercontent.com/resin-os/meta-resin/40d5a174da6b52d530c978e0cae22aa61f65d203/meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/update-resin-supervisor ; then
+            chmod 755 "/tmp/update-resin-supervisor"
+            PATH="/tmp:$PATH"
+            log "Added temporary supervisor updater replaced with fixed version..."
+        else
+            log WARN "Could not download temporary supervisor updater..."
```

Comment:
Do you not want it to error out in this case? Or at least enhance the warning saying this really needs to be fixed on the device afterwards?

Reply:
@floion yes, probably should, will update it like that. The stronger warning wouldn't make a difference, because logs from self-service updates will just stay on the device now, and won't get read by us, unless we need to see how things went.